### PR TITLE
[update] Upgrade Microcosm to 8.0

### PR DESCRIPTION
**Pull request by nhunzaker**
This commit upgrades Colonel Kurtz to the latest major release of Microcosm: 8.0.0.

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ module.exports = function(config) {
 
     logLevel: config.LOG_ERROR,
 
-    reporters: noCoverage ? [ 'spec' ] : [ 'spec', 'coverage' ],
+    reporters: noCoverage ? [ 'nyan' ] : [ 'nyan', 'coverage' ],
```
Comment (nhunzaker) [inline karma.conf.js]: I swear, beyond the nyan cat, the reporting is so good

File: src/stores/Blocks.js [hunk 2]
```diff
@@ -10,49 +10,48 @@
...
 
-module.exports = {
+let Blocks = {
+  register() {
+    return {
+      [Actions.create]  : this.create,
+      [Actions.destroy] : this.destroy,
+      [Actions.update]  : this.update,
+      [Actions.move]    : this.move
+    }
```
Comment (averyvery) [inline src/stores/Blocks.js]: Are these, like, default actions?

Comment (averyvery) [inline src/stores/Blocks.js]: or is this example code?

Comment (averyvery) [inline src/stores/Blocks.js]: NEVERMIND I THOUGHT I WAS LOOKING AT MICROCOSM. WRONG PROJECT

**Comment by averyvery** [comment]
:+1:

**Pull request merged**